### PR TITLE
Load Balancer EIP example

### PR DESCRIPTION
**Pull request by fg78nc**
Camel Spring Boot Load Balancer EIP example.

### CODE REVIEW

**Comment by fg78nc** [comment]
@oscerd Could you please review?

File: camel-example-spring-boot-load-balancer-eip/README.adoc [hunk 1]
```diff
@@ -0,0 +1,31 @@
+== Camel Example Spring Boot
+
+This example shows several examples of Load Balancer EIP with Apache Camel application using Spring Boot.
+
+1. Round-robin - The exchanges are selected from in a round robin fashion. This is a well known and classic policy, which spreads the load evenly.
```
Comment (oscerd) [inline camel-example-spring-boot-load-balancer-eip/README.adoc:5]: I think it should be only "in" and not "from in"

File: camel-example-spring-boot-load-balancer-eip/pom.xml [hunk 2]
```diff
@@ -0,0 +1,130 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (oscerd) [inline camel-example-spring-boot-load-balancer-eip/pom.xml:1]: License is missing

File: camel-example-spring-boot-load-balancer-eip/src/main/java/sample/camel/CustomLoadBalancer.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
+package sample.camel;
```
Comment (oscerd) [inline camel-example-spring-boot-load-balancer-eip/src/main/java/sample/camel/CustomLoadBalancer.java:1]: License is missing

Comment (oscerd) [inline camel-example-spring-boot-load-balancer-eip/src/main/java/sample/camel/CustomLoadBalancer.java:1]: Please use org.apache.camel.springboot.example as package

File: camel-example-spring-boot-load-balancer-eip/src/main/java/sample/camel/LoadBalancerEIPRouter.java [hunk 4]
```diff
@@ -0,0 +1,120 @@
+package sample.camel;
```
Comment (oscerd) [inline camel-example-spring-boot-load-balancer-eip/src/main/java/sample/camel/LoadBalancerEIPRouter.java:1]: License is missing and please use the package org.apache.camel.springboot.example

File: camel-example-spring-boot-load-balancer-eip/src/main/java/sample/camel/MyException.java [hunk 5]
```diff
@@ -0,0 +1,11 @@
+package sample.camel;
```
Comment (oscerd) [inline camel-example-spring-boot-load-balancer-eip/src/main/java/sample/camel/MyException.java:1]: Ditto

File: camel-example-spring-boot-load-balancer-eip/src/test/java/LoadBalancerEIPTest.java [hunk 6]
```diff
@@ -0,0 +1,170 @@
+package com.fg7;
```
Comment (oscerd) [inline camel-example-spring-boot-load-balancer-eip/src/test/java/LoadBalancerEIPTest.java:1]: Please use org.apache.camel.springboot.example and license is missing

**Review by oscerd — CHANGES_REQUESTED** [review]
Some findings.

File: camel-example-spring-boot-load-balancer-eip/README.adoc (hunk 1 above)
Comment (fg78nc) [reply camel-example-spring-boot-load-balancer-eip/README.adoc:5]: Description comes from https://camel.apache.org/components/latest/eips/loadBalance-eip.html
Do you want changes in both places?

Comment (oscerd) [reply camel-example-spring-boot-load-balancer-eip/README.adoc:5]: Sounds good

Comment (fg78nc) [reply camel-example-spring-boot-load-balancer-eip/README.adoc]: All requested changes are done, kindly verify.

**Review by oscerd — APPROVED** [review]

**Review by oscerd — APPROVED** [review]

**Pull request merged**